src/popup/pages/SeedPhraseVerifySettings.vue

```diff
@@ -54,12 +54,17 @@
         </div>
         <FixedScreenFooter>
           <BtnMain
+            v-if="hasError || !showNotification"
             class="verify-button"
-            :disabled="!selectedWordIds || selectedWordIds.length !== mnemonicShuffled.length"
+            :disabled="isVerifyButtonDisabled"
+            :text="$t('pages.seedPhrase.verify')"
             @click="verifyLastStep"
-          >
-            {{ $t('pages.seedPhrase.verify') }}
-          </BtnMain>
+          />
+          <BtnMain
+            v-else
+            :text="$t('pages.seedPhrase.goHome')"
+            @click="goHome"
```

Comment:
Buttons has the `to` property which makes them links which are more user friendly and easier to debug.